packages/client/src/components/app/charts/ApexChart.svelte

```diff
@@ -1,25 +1,82 @@
 <script>
   import { getContext } from "svelte"
-  import { chart } from "svelte-apexcharts"
-  import Placeholder from "../Placeholder.svelte"
+  import ApexCharts from "apexcharts"
+  import { Icon } from "@budibase/bbui"
+  import { cloneDeep } from "lodash"
```

Comment:
It isn't necessary to import lodash here. There is a simple `cloneDeep` in the `@budibase/bbui` helpers that should do the job.

Reply:
Replaced it with [structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) instead